react/features/base/tracks/middleware.js

```diff
@@ -72,6 +73,11 @@ MiddlewareRegistry.register(store => next => action => {
             store.dispatch(getAvailableDevices());
         }
 
+        if (getSourceNameSignalingFeatureFlag(store.getState())
+            && action?.track?.jitsiTrack.videoType === VIDEO_TYPE.DESKTOP) {
```

Comment:
can action or track be undefined? seems to be overly defensive, it's better to crash